cibuildwheel/__main__.py

```diff
@@ -136,16 +136,17 @@ def main():
     # This needs to be passed on to the docker container in linux.py
     os.environ['CIBUILDWHEEL'] = '1'
 
-    if not os.path.exists(os.path.join(project_dir, 'setup.py')):
-        print('cibuildwheel: Could not find setup.py at root of project', file=sys.stderr)
+    if not os.path.exists(os.path.join(package_dir, 'setup.py')):
+        print('cibuildwheel: Could not find setup.py at root of package', file=sys.stderr)
         exit(2)
 
     if args.print_build_identifiers:
         print_build_identifiers(platform, build_selector)
         exit(0)
 
     build_options = dict(
-        project_dir=project_dir,
+        project_dir='.',
```

Comment:
What about calling `os.getcwd()` here?

Reply:
Or even better: killing the `project_dir` parameter, and just calling `os.getcwd()` in `linux.py`/`macos.py`/`windows.py` ?

Reply:
I left `project_dir` because it still allows a bit advanced configuration if needed. But I can update it to `os.getcwd()`.